pkg/common/rotationutil/rotationutil_test.go

```diff
@@ -86,3 +95,13 @@ func TestJWTSVIDExpiresSoon(t *testing.T) {
 
 	assert.True(t, JWTSVIDExpiresSoon(expiredJWT, mockClk.Now()))
 }
+
+// halfLife(beginTime, expiryTime time.Time)
+func TestHalfLife(t *testing.T) {
```

Comment:
I'm wondering if we really want to have this test function. Ideally, we should not need to test non-exported functions (that usually leads to fragile tests), but rather cover them through test cases of the exported function. Are there more improvements that we could do in the test ShouldRotateX509 to make sure that a proper jitter is added, within the ranges that we expect?